tests/SignedAuthenticationUrlTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+use Creads\Partners\SignedAuthenticationUrl;
+use PHPUnit\Framework\TestCase;
+
+final class SignedAuthenticationUrlTest extends TestCase
+{
+
+    //Client APP_ID: 15_64d081gwqckkwwkc0s0occ4ckss8ww04o4ow80k0444kkkg4g0
+    //Client SECRET_KEY: 27ldyvaorh34coc00kggkogscgwc8s84go8ck8wks4ogsgg8kk
+    //bin/partners signed-auth-url  "My Organization" j.doe@creads.org THIS-IS-MY-REMOTE-ID_000 John Doe -v
```

Comment:
Actually, the first parameters is the user email, not the organization name. Like this, the URL could be well generated and validated but sounds not well formatted.

Reply:
I fixed the tests to have encoded parameters in the right place